backend/src/main/kotlin/com/example/backend/utils/CalculadoraPrimos.kt

```diff
@@ -0,0 +1,29 @@
+package com.example.backend.utils
+
+class CalculadoraPrimos {
+    companion object {
```

Comment:
```suggestion
object CalculadoraPrimos {
```
No Kotlin também existe o [`object`](https://kotlinlang.org/docs/object-declarations.html#object-declarations-overview), que pode ser utilizado quando todos os membros são estáticos